services/upload/src/main/java/app/coronawarn/server/services/federation/upload/runner/TestDataGeneration.java

```diff
@@ -77,7 +82,7 @@ private DiagnosisKey makeKeyFromTimestamp(long timestamp) {
         .withSubmissionTimestamp(timestamp)
         .withVisitedCountries(List.of("FR", "DK"))
```

Comment:
shouldn't the list of countries be configurable or use the list of available countries?

Reply:
The configuration for supported countries was done in distribution and submission, but not in upload. So we don't have this information here.